packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -209,7 +209,7 @@ class Draggable<T extends Object> extends StatefulWidget {
     this.feedbackOffset = Offset.zero,
     @Deprecated(
       'Use dragAnchorStrategy instead. '
-      'This feature was deprecated after 1.27.0-8.0.pre'
+      'This feature was deprecated after 2.1.0-10.0.pre.'
```

Comment:
The analyzer is failing due to a missing 'v' in these notices. 😅 
```suggestion
      'This feature was deprecated after v2.1.0-10.0.pre.'
```